cnv/cnv_virtual_machines/cnv-create-vms.adoc

```diff
@@ -4,6 +4,12 @@ include::modules/cnv-document-attributes.adoc[]
 :context: cnv-create-vms
 toc::[]
 
+.Prerequisites
```

Comment:
This should be an admonition, not a prerequisite, and since it can lead to loss of service it should probably be a warning admonition:
[WARNING]
====
Do not create virtual machines in `openshift-*` namespaces. 
Create a new namespace or use an existing namespace that does not have the `openshift-*` prefix.
====

And for layout reasons, I'd move it after the 'Use one of these procedures to...' list

Reply:
Cool, thanks. Had some strange formatting problems in Atom, where the * was not rendering.

Warning reads: 
[WARNING]
====
Do not create virtual machines in `openshift-*` namespaces.
Instead, create a new namespace or use an existing namespace without the
`openshift` prefix.
====